src/Compilers/CSharp/Portable/Symbols/OverriddenOrHiddenMembersHelpers.cs

```diff
@@ -165,6 +168,18 @@ private static OverriddenOrHiddenMembersResult MakeOverriddenOrHiddenMembersWork
             return OverriddenOrHiddenMembersResult.Create(overriddenMembers, hiddenMembers, runtimeOverriddenMembers);
         }
 
+        /// <summary>
+        /// Compute a candidate overridden method when a method knows what method it is intended to
+        /// override. This makes a particular difference when covariant returns are used, in which
+        /// case the signature matching rules would not compute the correct overridden method.
+        /// </summary>
+        private static MethodSymbol KnownOverriddenClassMethod(MethodSymbol method) => method switch
```

Comment:
Nit: I'd prefer this to be on the next line and indented.

Reply:
Are you suggesting a newline after the arrow?

---
In reply to: [445847240](https://github.com/dotnet/roslyn/pull/44025#discussion_r445847240) [](ancestors = 445847240)

Reply:
I find it looks best with the arrow on the next line entirely, but after the arrow would also work. The way it's formatted right now, though, the switch cases braces look like a method body to a quick scan.

---
In reply to: [445849322](https://github.com/dotnet/roslyn/pull/44025#discussion_r445849322) [](ancestors = 445849322,445847240)